### DISCUSSION TITLE
How to connect Two links ?

### DISCUSSION THREAD

**Discussion by Lijesh-km** [clientIO/joint#1986] [Q&A] [answered]
is it possible to connect two links like this?
![w](https://user-images.githubusercontent.com/121955632/212143278-9cc980ea-6e3a-4a43-953d-47dc182dac1f.png)
![test](https://user-images.githubusercontent.com/121955632/212143419-f4db277c-fd27-46fe-a15b-38884deeb762.png)

I need to connect one or more links to one link

**Comment by kumilingus** [comment, marked as answer]
Do you mean to allow the users to connect links to links via UI?

Connecting links to links is disabled by default. To allow this use [validateConnection()](https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#dia.Paper.prototype.options.validateConnection) paper option callback.

Check out [sequence diagram](https://www.jointjs.com/demos/sequence-diagram) example.

**Reply by Lijesh-k** [reply]
@kumilingus 
Yes, need to connect two links via  UI. How can I do it?

**Reply by kumilingus** [reply]
> Connecting links to links is disabled by default. To allow this use [validateConnection()](https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#dia.Paper.prototype.options.validateConnection) paper option callback.
Check out [sequence diagram](https://www.jointjs.com/demos/sequence-diagram) example.

**Reply by Lijesh-k** [reply]
Thanks, @kumilingus 
The link to the link connection working now, But the link is connecting to the center of the link how can I avoid that 
![link](https://user-images.githubusercontent.com/122600888/212458040-59d8b257-442b-4585-8fd2-963e5e4473a3.png)

**Reply by kumilingus** [reply]
```js
paper.options.connectionStrategy = joint.connectionStrategies.pinAbsolute;
```
To find out more, please read the [docs](https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#connectionStrategies.intro).
See *sequence* demo [source code](https://github.com/clientIO/joint/blob/master/demo/sequence/src/sequence.js#L50-L70) to see an advanced use of `connectionStrategy`.

**Reply by Lijesh-k** [reply]
Thank you @kumilingus
